magicbyirineu/Modules/CardDetail/View/CardDetailViewController.swift

```diff
@@ -39,5 +39,12 @@ class CardDetailViewController: UIViewController {
         self.dismiss(animated: true, completion: nil)
     }
     
+}
 
+extension CardDetailViewController{
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
```

Comment:
Acho que fica melhor sem extension aqui. 

Use extension quando realmente for uma extension. Aqui está apenas separando um pedaço de código da qual realmente pertence a controller.

Reply:
Feito, vide Commit mais recente :)